cogs/covidcog.py

```diff
@@ -0,0 +1,58 @@
+import discord
+from discord.ext import commands
+import requests
+
+
+def embedCreator(data):
+    state = data["state"]
+    embedTitle = ":world_map: Covid Stats in " + state + " :world_map:"
+    firstField = {
+        "name": "Total " + state + " Cases",
+        "value": "Total Cases: {:,}".format(data["cases"]),
+        "inline": True
+    }
+    secondField = {
+        "name": "Total " + state + " Deaths",
+        "value": "Total Deaths: {:,}".format(data["deaths"]),
+        "inline": True
+    }
+    thirdField = {
+        "name": "Total " + state + " Tests",
+        "value": "Total Tests: {:,}".format(data["tests"]),
+        "inline": True
+    }
+    embed = discord.Embed(title=embedTitle, color=0x8D0000)
+    embed.description = "Statistics for Covid-19 in " + state + "\n"
+    embed.add_field(
+        name=firstField["name"], value=firstField["value"], inline=firstField["inline"])
```

Comment:
You can split the method call over lines, so creating the dictionaries is a little redundant.

E.g.
```python
embed.add_field(
    name=f"Total {state} Deaths",
    value=f"Total Deaths: {data['deaths']}",
    inline=True
)
```